.github/workflows/docs.yml

```diff
@@ -1,52 +1,25 @@
 name: Docs
 
 on:
-  # Build docs on pull request to verify changes
```

Comment:
I think it is simpler to move the "building" of the docs to the verify-docker-image job which should actually just be called the "ci" job or the "pull request" job. It is "everything" that runs on a pull request and uses a single build to run all tests and produce all artifacts.

Reply:
Same rationale for extract locales. We "could" solve this by no specifying either version or digest and forcing the run action to build the image but this will build using a different mechanism than our "build-docker" job so not 100% guaranteed it will produce the same result. That's like a huge mega edge case though so happy to hear push back.

Reply:
I agree - we should eventually rename that workflow, but for now this is fine.